pype/plugins/standalonepublisher/publish/extract_trim_video_audio.py

```diff
@@ -60,7 +60,7 @@ def process(self, instance):
                 ]
 
             args = [
-                ffmpeg_path,
+                f"\"{ffmpeg_path}\"",
```

Comment:
WAIT

Reply:
This is Python 3. That would not work in hiero!

Reply:
this is standalone publisher

Reply:
Damn why do we have same plugin name in standalone publisher and in global...